packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -893,8 +896,8 @@ class MemoryBodyState extends State<MemoryBody>
   //                final displayData = encoder.convert(data);
   String longValueToShort(String longValue) {
     var value = longValue;
-    if (longValue.length > 35) {
-      final firstPart = longValue.substring(0, 10);
+    if (longValue.length > 34) {
+      final firstPart = longValue.substring(0, 9);
```

Comment:
can you put 34 and 9 into named consts so we know what they are for?

Reply:
Done.